var/spack/repos/builtin/packages/totalview/package.py

```diff
@@ -32,6 +32,8 @@ class Totalview(Package):
     # as resources dependent on the specific architecture used.
     version('2019.2.18',
             sha256='09e5c554032af945f8cf147dd548421267e50e906cc9686fb5cd0e8e63fcf650')
+    version('2021.1.16',
+            sha256='4c51c7b6ab6b6afa7635ba2e9fc3b0ef833806f775a0ad0da26b13d6320625dd')
```

Comment:
New versions are added to the top of the version directive section:

```suggestion
    version('2021.1.16',
            sha256='4c51c7b6ab6b6afa7635ba2e9fc3b0ef833806f775a0ad0da26b13d6320625dd')
    version('2019.2.18',
            sha256='09e5c554032af945f8cf147dd548421267e50e906cc9686fb5cd0e8e63fcf650')
```

Reply:
I cannot readily confirm the sha256.